modules/Configurator/language/en_us.lang.php

```diff
@@ -130,45 +130,46 @@
-    'SYSTEM_NAME' => 'System Name:',
-    'SYSTEM_NAME_WIZARD' => 'Name:',
-    'SYSTEM_NAME_HELP' => 'This is the name that displays in the title bar of your browser.',
-    'LBL_LDAP_TITLE' => 'LDAP Authentication Support',
-    'LBL_LDAP_ENABLE' => 'Enable LDAP',
-    'LBL_LDAP_SERVER_HOSTNAME' => 'Server:',
-    'LBL_LDAP_SERVER_PORT' => 'Port Number:',
-    'LBL_LDAP_ADMIN_USER' => 'User Name:',
-    'LBL_LDAP_ADMIN_USER_DESC' => 'Used to search for the SuiteCRM user. [May need to be fully qualified] It will bind anonymously if not provided.',
-    'LBL_LDAP_ADMIN_PASSWORD' => 'Password:',
-    'LBL_LDAP_AUTHENTICATION' => 'Authentication:',
-    'LBL_LDAP_AUTHENTICATION_DESC' => 'Bind to the LDAP server using a specific users credentials',
-    'LBL_LDAP_AUTO_CREATE_USERS' => 'Auto Create Users:',
-    'LBL_LDAP_USER_DN' => 'User DN:',
-    'LBL_LDAP_GROUP_DN' => 'Group DN:',
-    'LBL_LDAP_GROUP_DN_DESC' => 'Example: <em>ou=groups,dc=example,dc=com</em>',
-    'LBL_LDAP_USER_FILTER' => 'User Filter:',
-    'LBL_LDAP_GROUP_MEMBERSHIP' => 'Group Membership:',
-    'LBL_LDAP_GROUP_MEMBERSHIP_DESC' => 'Users must be a member of a specific group',
-    'LBL_LDAP_GROUP_USER_ATTR' => 'User Attribute:',
-    'LBL_LDAP_GROUP_USER_ATTR_DESC' => 'The unique identifier of the person that will be used to check if they are a member of the group Example: <em>uid</em>',
-    'LBL_LDAP_GROUP_ATTR_DESC' => 'The attribute of the Group that will be used to filter against the User Attribute Example: <em>memberUid</em>',
-    'LBL_LDAP_GROUP_ATTR' => 'Group Attribute:',
-    'LBL_LDAP_USER_FILTER_DESC' => 'Any additional filter params to apply when authenticating users e.g.<em>is_suitecrm_user=1 or (is_suitecrm_user=1)(is_sales=1)</em>',
-    'LBL_LDAP_LOGIN_ATTRIBUTE' => 'Login Attribute:',
-    'LBL_LDAP_BIND_ATTRIBUTE' => 'Bind Attribute:',
-    'LBL_LDAP_BIND_ATTRIBUTE_DESC' => 'For Binding the LDAP User Examples:[<b>AD:</b>&nbsp;userPrincipalName] [<b>openLDAP:</b>&nbsp;userPrincipalName] [<b>Mac&nbsp;OS&nbsp;X:</b>&nbsp;uid] ',
-    'LBL_LDAP_LOGIN_ATTRIBUTE_DESC' => 'For searching for the LDAP User Examples:[<b>AD:</b>&nbsp;userPrincipalName] [<b>openLDAP:</b>&nbsp;dn] [<b>Mac&nbsp;OS&nbsp;X:</b>&nbsp;dn] ',
-    'LBL_LDAP_SERVER_HOSTNAME_DESC' => 'Example: ldap.example.com or ldaps://ldap.example.com for SSL',
-    'LBL_LDAP_SERVER_PORT_DESC' => 'Example: <em>389 or 636 for SSL</em>',
-    'LBL_LDAP_GROUP_NAME' => 'Group Name:',
-    'LBL_LDAP_GROUP_NAME_DESC' => 'Example <em>cn=suitecrm</em>',
-    'LBL_LDAP_USER_DN_DESC' => 'Example: <em>ou=people,dc=example,dc=com</em>',
-    'LBL_LDAP_AUTO_CREATE_USERS_DESC' => 'If an authenticated user does not exist one will be created.',
-    'LBL_LDAP_ENC_KEY' => 'Encryption Key:',
-    'DEVELOPER_MODE' => 'Developer Mode',
-
-    'SHOW_DOWNLOADS_TAB' => 'Display Downloads Tab',
-    'LBL_LDAP_ENC_KEY_DESC' => 'For SOAP authentication when using LDAP.',
+	'SYSTEM_NAME'=>'System Name:',
```

Comment:
Isn't this new indentation and no space in '=>' a step back from the format already in use?
@Dillon-Brown Can you please reformat this file?